apps/web/lib/features/daily-plan/add-task-estimation-hours-modal.tsx

```diff
@@ -678,20 +701,30 @@ function TaskCard(props: ITaskCardProps) {
 		try {
 			setAddToPlanLoading(true);
 
-			console.log(plan);
-
-			if (plan.id) {
+			if (plan && plan.id) {
 				await addTaskToPlan({ taskId: task.id }, plan.id);
 			} else {
-				await createDailyPlan({
-					workTimePlanned: 0,
-					taskId: task.id,
-					date: moment(plan.date).toDate(),
-					status: DailyPlanStatusEnum.OPEN,
-					tenantId: user?.tenantId ?? '',
-					employeeId: user?.employee.id,
-					organizationId: user?.employee.organizationId
-				});
+				if (plan) {
```

Comment:
@CREDO23 not a good code, why you can't just create var like "let somedate" and set value of it depending on those if statements and below just do one call to 
```
await createDailyPlan({
						workTimePlanned: 0,
						taskId: task.id,
						date: someday, <------
						status: DailyPlanStatusEnum.OPEN,
						tenantId: user?.tenantId ?? '',
						employeeId: user?.employee.id,
						organizationId: user?.employee.organizationId
					});
```
This is basic, no?

Reply:
@CREDO23 I am merging for now, please improve that in another PR etc